.github/workflows/ci.yml

```
@@ -26,35 +30,38 @@ jobs:
         with:
           file: ./backend/coverage.xml
 
-  frontend:
+  frontend-test:
     runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        shard: [1, 2]
     steps:
       - uses: actions/checkout@v3
```

actionlint (1.7.12)

[error] the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)